### DISCUSSION TITLE
Debian 11 ---->>> E0413 02:18:01.018146    2329 memcache.go:287] couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request

### DISCUSSION THREAD

**Discussion by betashepherd** [k3s-io/k3s#7411] [Q&A] [answered]
```console
root@localhost:~# curl -sfL https://get.k3s.io | INSTALL_K3S_CHANNEL=stable INSTALL_K3S_VERSION=v1.26.3+k3s1 INSTALL_K3S_EXEC="--disable=traefik" sh -
[INFO]  Using v1.26.3+k3s1 as release
[INFO]  Downloading hash https://github.com/k3s-io/k3s/releases/download/v1.26.3+k3s1/sha256sum-amd64.txt
[INFO]  Downloading binary https://github.com/k3s-io/k3s/releases/download/v1.26.3+k3s1/k3s
[INFO]  Verifying binary download
[INFO]  Installing k3s to /usr/local/bin/k3s
[INFO]  Skipping installation of SELinux RPM
[INFO]  Creating /usr/local/bin/kubectl symlink to k3s
[INFO]  Creating /usr/local/bin/crictl symlink to k3s
[INFO]  Creating /usr/local/bin/ctr symlink to k3s
[INFO]  Creating killall script /usr/local/bin/k3s-killall.sh
[INFO]  Creating uninstall script /usr/local/bin/k3s-uninstall.sh
[INFO]  env: Creating environment file /etc/systemd/system/k3s.service.env
[INFO]  systemd: Creating service file /etc/systemd/system/k3s.service
[INFO]  systemd: Enabling k3s unit
Created symlink /etc/systemd/system/multi-user.target.wants/k3s.service → /etc/systemd/system/k3s.service.
/usr/sbin/iptables-restore
/usr/sbin/iptables-save
/usr/sbin/ip6tables-restore
/usr/sbin/ip6tables-save
[INFO]  systemd: Starting k3s
root@localhost:~# kubectl get pod
E0413 02:18:01.018146    2329 memcache.go:287] couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
E0413 02:18:01.023545    2329 memcache.go:121] couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
E0413 02:18:01.033053    2329 memcache.go:121] couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
E0413 02:18:01.037175    2329 memcache.go:121] couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
```

**Comment by hudeng-go** [comment]
use journalctl -u k3s get k3s server log?

**Comment by brandond** [comment, marked as answer]
Ensure that you have the required ports and IP ranges open as per https://docs.k3s.io/advanced#ubuntu

**Reply by brandond** [reply]
@betashepherd please confirm that you have disabled any iptables-based firewalls or opened the correct ports and IP ranges as described in the documentation. The symptoms you're showing are almost always caused by firewall rules blocking CNI traffic.

**Comment by betashepherd** [comment]
`
root@localhost:~# cat /etc/issue
Debian GNU/Linux 11 \n \l

root@localhost:~# uname -r
5.10.0-21-amd64
root@localhost:~# uname -a
Linux localhost 5.10.0-21-amd64 #1 SMP Debian 5.10.162-1 (2023-01-21) x86_64 GNU/Linux
`

**Comment by betashepherd** [comment]
use journalctl -u k3s get k3s server  @hudeng-go 

[k3s.log](https://github.com/k3s-io/k3s/files/11227978/k3s.log)

**Comment by betashepherd** [comment]
root@localhost:~# k3s --version
k3s version v1.26.3+k3s1 (01ea3ff2)
go version go1.19.7

**Comment by betashepherd** [comment]
<img width="1122" alt="截屏2023-04-14 09 54 17" src="https://user-images.githubusercontent.com/5636922/231921410-d5eb3be8-e99e-4c04-86d8-340f36826dcc.png">

**Comment by betashepherd** [comment]
> Ensure that you have the required ports and IP ranges open as per https://docs.k3s.io/advanced#ubuntu

I use this cmd to install on Debian 11 online

`curl -sfL https://get.k3s.io | INSTALL_K3S_CHANNEL=stable INSTALL_K3S_EXEC="--disable=traefik" sh -`

**Comment by betashepherd** [comment]
<img width="1118" alt="截屏2023-04-14 09 58 58" src="https://user-images.githubusercontent.com/5636922/231922442-bbc2b3c0-f443-42a7-835c-a10cc17a9617.png">

**Comment by betashepherd** [comment]
<img width="1306" alt="截屏2023-04-14 10 00 25" src="https://user-images.githubusercontent.com/5636922/231922645-91a2f899-4527-439d-a5c3-98066cdd73af.png">
